### ISSUE TITLE
Analytics → Settings results in blank screen

### ISSUE THREAD

**Issue by Luminus** [woocommerce/woocommerce#29506]
**Prerequisites (mark completed items with an [x]):**
- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**
It is not possible to access the Settings page under the Analytics menu. It shows a blank screen and there's a `Minified React error #185` in the console.

Per the React docs, the full text of the error is as follows:

Maximum update depth exceeded. This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. React limits the number of nested updates to prevent infinite loops.

https://reactjs.org/docs/error-decoder.html/?invariant=185

**Expected behavior**
The Settings page should load and not cause any errors

**Actual behavior**
We're greeted with a blank page instead of the settings page

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:
1. Go to **Analytics → Settings**
2. You'll be greeted with a blank screen
3. Open the browser Console

**Screenshots**
Error screenshot:
![Minified React Error 185](https://user-images.githubusercontent.com/248077/112643005-ca1c5080-8e3b-11eb-9eac-a7ebbbf343e8.png)

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
We use the [WooCommerce System Status Report](https://docs.woocommerce.com/document/understanding-the-woocommerce-system-status-report/) to help us evaluate the issue. 
Without this report we won't be able to fully evaluate this issue.
<details>
```
The System Status Report is found in your WordPress admin under **WooCommerce > Status**. 
Please select “Get system report”, then “Copy for support”, and then paste it here.
```
</details>

**Comment by tammullen** [comment]
Thanks @Luminus for raising this! 

Analytics is being developed in another repository: https://github.com/woocommerce/woocommerce-admin.

This issue has already been raised there and a fix should be available in WooCommerce 5.2.
woocommerce/woocommerce-admin#6549

**Comment by Luminus** [comment]
Thanks for the heads up @tammullen. Much appreciated
